common.sh

```diff
@@ -390,12 +391,27 @@ if [[ ! -z ${AGENT_E2E_TEST_SCENARIO} ]]; then
   export IP_STACK=$(echo ${arr[1]##*IP} | tr V v)
   
   if [[ "$delimiterCount" == "2" ]]; then
-    export NETWORKING_MODE=${arr[2]}
-    if [[ $NETWORKING_MODE != "DHCP" ]]; then
+    val=${arr[2]}
+    if [[ "${val}" == "DHCP" ]]; then
+      export NETWORKING_MODE=$val
+    elif [[ "${val}" == "PXE" ]]; then
```

Comment:
I think we want to use/redefine the values to `SNO_IPV4_DHCP_ISO` instead of the current `SNO_IPV4_DHCP` when creating the ISO. This way, it will be easy to parse the `BOOT_MODE` to either `ISO` or `PXE` and will also help in readability.

e.g. `SNO_IPV4_DHCP_ISO`, `SNO_IPV4_DHCP_PXE`

Reply:
That is going to break all the CI jobs, if we don't have it defaulted

Reply:
It defaults to an empty string [here](https://github.com/openshift-metal3/dev-scripts/pull/1526/files#diff-a045819fa24dfbeb74522ddaf6eac77b5a4d39762752db45251e6101cd5c987dR379) but I agree on an explicit value as `ISO` will be helpful.